changelog.txt

```diff
@@ -1,3 +1,12 @@
+4.9.0
+============
+New Features:
+**Broker support for Xamarin Android**. MSAL.NET now supports brokered authentication with Xamarin Android. For details see https:aka.ms/msal-net-brokers, along with code snippets, and more details in the [4.9 release blog post](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/msal-net-4.9#broker-support-on-xamarinandroid). For help migrating from ADAL.NET using iOS broker to MSAL.NET using android broker, see [this page on migration](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/How-to-migrate-from-using-Android-Broker-on-ADAL.NET-to-MSAL.NET).
```

Comment:
This page does not exist yet. @trwalke are you going to add it?
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/How-to-migrate-from-using-Android-Broker-on-ADAL.NET-to-MSAL.NET